lib/checker.js

```diff
@@ -1673,6 +1673,23 @@ function typesMatch(type1,type2) {
 	);
 }
 
+function isAssignmentAllowed(type1, type2){
+	if(typesMatch(type1, type2)){
+		return true;
+	}
+
```

Comment:
I think we could merge the `any` branch into this one, and insert like `if (type1 == "any") return true;` here, and then take out the [`if` statement here](https://github.com/getify/Typl/pull/25/files#diff-605f23c7e3628b173a428e6a469e6ff6R1207).

Do you think that would make sense?

Reply:
:100: %